.gitignore

```diff
@@ -1,2 +1,2 @@
 # Dependency directories
-node_modules/
+# node_modules/
```

Comment:
What did this line of code do?

Reply:
I comment this line out, just wanted to make sure that all the dependencies that we were installing in our project are being tracked.

Reply:
Glad we got this resolved!